internal/resources/fetcher.go

```diff
@@ -88,8 +151,29 @@ func (f *Fetcher) getResourcesDynamically(ctx context.Context,
 		Version:  version,
 		Resource: resource,
 	}
-	list, err := f.dynamicClient.Resource(resourceID).Namespace(namespace).
-		List(ctx, metav1.ListOptions{})
+	var list *unstructured.UnstructuredList
+	var err error
+	// if no namespace, we need to look for cluster wide resources
```

Comment:
I think this comment is a leftover from the initial PR. I think it can be removed now